.github/workflows/ui-tests.yml

```
@@ -65,6 +66,7 @@ jobs:
           path: ./packages/ui/coverage/lcov.info
           retention-days: 1
 
+    timeout-minutes: 15
```

actionlint (1.7.12)

[error] key "timeout-minutes" is duplicated in "test" job. previously defined at line:17,col:5

(syntax-check)